### PR TITLE
Iae 35523/cards

### PR DESCRIPTION
**Pull request by ascreven-gsa**
- Fixed card color array issue by adding demo-card component that calls in actual card component
- Card rendered in Landing Page component
- Fixed border/border radius issue

Checked in Chrome, Firefox, Safari, Edge, and IE 
*IE borders look a little weird in the Fractal layout but confirmed it's not an issue in the full-screen view

### CODE REVIEW

File: src/stylesheets/components/_card.scss [hunk 1]
```diff
@@ -44,13 +45,15 @@
     }
 
     &--left, &--center {
+      svg {
+        @include u-margin-right(1);
+      }
       > * {
         display: inline-block;
```
Comment (AlejandroColinaGSA) [inline src/stylesheets/components/_card.scss:52]: Use uswds utitlities `@include u-display('inline-block')` everywhere, there are several of these throughout the file. Note: I personally do not see the utility of this but it is a rule.

File: src/stylesheets/components/_card.scss [hunk 2]
```diff
@@ -44,13 +45,15 @@
...
         vertical-align: bottom;
       }
     }
 
-
     &--stacked, &--center {
       text-align: center;
```
Comment (AlejandroColinaGSA) [inline src/stylesheets/components/_card.scss:58]: Just another example of 'please, replace with USWDS utility'.  There is more but I won't make a comment for each

File: src/stylesheets/components/_card.scss [hunk 3]
```diff
@@ -88,13 +91,22 @@
...
+    @include u-border('base-light', 'solid', $sds-card-border-width);
+    border-radius: $sds-card-border-radius;
+    .sds-card__header {
+      border-top-left-radius: $inner-border-radius;
+      border-top-right-radius: $inner-border-radius;
+    }
   }
 
   &--shadowed {
     box-shadow: 0 6px 12px 0 rgba(0, 0, 0, 0.1);
```
Comment (AlejandroColinaGSA) [inline src/stylesheets/components/_card.scss:103]: use uswds shadow units

File: src/components/03-components/cards/_card.njk [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+			{% endif -%}
+
+		</div>
+	{%- endif -%}
+
+	<div class="sds-card__content{% if body.grid %} grid-row{% endif %}">
+
+		{%- if body.lists -%}
+			{%- if body.grid %}
+				<div class="grid-col">
```
[the remark this answers is not in the document]
Comment (ascreven-gsa) [reply src/components/03-components/cards/_card.njk:46]: Cards should have a set structure and shouldn't be used as sections as sections 
https://designsystem.digital.gov/components/card/

File: src/components/03-components/cards/cards-demo.config.yml [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+    context:
+      card:
+        header:
+          buttons:
+            - modifiers:
+              - outline
+          style: flex
+  - name: header buttons
+    label: Header with Two Buttons
+    notes: A horizontal card header may have a title and two buttons.
```
[the remark this answers is not in the document]
Comment (ascreven-gsa) [reply src/components/03-components/cards/cards-demo.config.yml:116]: I based the variants off Christy's specs

[the remark this answers is not in the document]
Comment (AlejandroColinaGSA) [reply src/components/03-components/cards/cards-demo.config.yml:116]: I suppose that the problem is that there is semantic "cards" and there is "things-with-borders-and-rounded-corners-and-shadows" that we could reuse. For example page sections, or pop-ups, should we use the card class?

**Comment by ascreven-gsa** [comment]
> I noticed that there are two config files is this correct?
> when looking at the website the cards are under a "demo" category. please remove that

Necessary to nest cards in Sam Styles while keeping all color and structure variations

**Pull request merged**